Source/CarthageKit/Git.swift

```diff
@@ -269,16 +269,16 @@ private func parseConfigEntries(_ contents: String, keyPrefix: String = "", keyS
 	}
 }
 
-/// Git’s representation of file system objects at a path relative to the repository root.
+/// Lists the contents of the directory at a given path relative to the repository root.
 ///
-/// - parameter path: Path separators at the end of `path` have significance of outputting directory contents.
```

Comment:
Double checking: this comment is not relevant anymore because of the normalization done by

```
let directoryPath = URL(fileURLWithPath: path, isDirectory: true).relativeString
```

?

Why `relativeString`? There is no baseURL, so this will default to `absoluteString`

Reply:
That's why I deleted the comment about the path parameter.

Running a `URL(fileURLWithPath: "Carthage/Checkouts", isDirectory: true).absoluteString` in a playground will give you some weird thing like `"file:///private/var/folders/3h/7fj4dcs97lqf0yc46r8c64yc0000gn/T/com.apple.dt.Xcode.pg/containers/com.apple.dt.playground.stub.iOS_Simulator.MyPlayground3-33380BCE-E588-450F-8170-5C1FADA1EF1E/Carthage/Checkouts/"`. I think it is actually relative to the current directory, so we can't use `absoluteString`.

Reply:
I’m wondering if I have something to add .. or forward . Let me know thanks

Reply:
@CosynPa what is the output with `relativeString` instead?

Reply:
"Carthage/Checkouts/". The key point here is to make sure there is a slash at the end.

Reply:
ok! Thanks!